src/Symfony/Component/HttpFoundation/Session/Storage/Handler/AbstractSessionHandler.php

```diff
@@ -32,6 +32,9 @@
     public function open($savePath, $sessionName)
     {
         $this->sessionName = $sessionName;
+        if (!headers_sent() && !ini_get('session.cache_limiter')) {
```

Comment:
Shouldn't we raise a warning or something in `!ini_get('session.cache_limiter') && headers_sent()` ?

Reply:
not sure, there are many similar checks in the code base already

Reply:
Well, [this one](https://github.com/symfony/symfony/blob/9f1c017983a281ad4f29b8681fda1530d4e750c1/src/Symfony/Component/HttpFoundation/Response.php#L322) convinced me... 😅